tests/unit/js/html-test-data.js

```diff
@@ -224,3 +224,88 @@ export const readingLogDropperForm = `
     </form>
 </div>
 `
+export const readClassification = `
+<fieldset class="major" id="classifications"  data-config="{&quot;Please select a classification.&quot;: &quot;Por favor, selecione uma classificação.&quot;, &quot;You need to give a value to CLASS.&quot;: &quot;Você precisa dar um valor ao CLASS.&quot;}" >
+    <legend>Classificações</legend>
+    <div class="formBack">
+
+        <div id="classification-errors" class="note" style="display: none"></div>
+        <div class="formElement">
+            <div class="label">
+                <label for="select-classification">Você conhece alguma classificação para essa edição?</label>
+                <span class="tip">Como Decimal Dewey?</span>
+            </div>
+            <div class="input">
+                <table class="classifications identifiers">
+                    <tbody><tr id="classifications-form">
+                        <td align="right">
+                            <input type="hidden" name="select-classification-json" class="repeat-ignore" value="[]"><select name="name" id="select-classification">
+                                <option value="">Selecione um de muitos...</option>
+                                <option value="dewey_decimal_class">Dewey Decimal Class</option>
+
+                                <option value="lc_classifications">Library of Congress</option>
+
+                                <option value="library_and_archives_canada_cataloguing_in_publication">Library and Archives Canada Cataloguing in Publication</option>
+
+                                <option value="library_bibliographical_classification">Library-Bibliographical Classification</option>
+
+                                <option value="rvk">Regensburger Verbundklassifikation</option>
+
+                                <option value="finnish_public_libraries_classification_system">Finnish Public Libraries</option>
+
+                                <option value="udc">Universal Decimal Classification</option>
+
+                                <option value="ulrls_classmark">ULRLS Classmark</option>
+
+                                <option value="goethe_university_library,_frankfurt">Goethe University Library, Frankfurt</option>
+
+                                <option value="siso">SISO</option>
+
+                                <option value="nur">NUR</option>
+
+                                <option value="identificativo_sbn">Identificativo SBN</option>
+
+                                <option value="---">---</option>
+
+                                <!-- <option value="__add__">Adicionar um novo tipo de classificação</option> -->
```

Comment:
I am curious about `<option value="---">---</option>`. Is that simply how things are for you? Where were you looking when you saw this? I am wondering if I am missing some way this gets toggled depending on the type of user, or locale, or something. 

For me, on both production and in the development environment, I have:
```html
<!-- <option>---</option> -->
<!-- <option value="__add__">Add a new classification type</option> -->
```

Reply:
I perfectly understand the situation. I removed this HTML directly from the browser and removed the comment from line 268 to make the test of the functionality, because the functionality owned the condittion of this value.
I don't speake english very well, so, iam sorry for any mistakes on my speak.